frontend/src/app/note/note.component.ts

```diff
@@ -79,10 +87,11 @@ export class NoteComponent implements AfterContentChecked {
     this.editionDone.emit({id: this.note.id, text: this.noteForm.value.text});
   }
 
-  ngAfterContentChecked(): void {
-    if (this.shouldGiveFocus && this.textArea && this.textArea.nativeElement) {
-      this.textArea.nativeElement.focus();
-      this.shouldGiveFocus = false;
-    }
+  ngAfterViewInit() {
+    concat(of(this.textAreaQuery), this.textAreaQuery.changes).pipe(
+      filter(() => this.textAreaQuery.length === 1)
```

Comment:
@cexbrayat I don't get why `QueryList.changes` is an `Observable<any>`. If I add a console log, it seems that it actually emits itself, so I guess it could be typed as `Observable<QueryList<T>>`.

Reply:
Yes it emits itself. I don't know why it is typed with any...